specification.markdown

```diff
@@ -92,9 +92,9 @@ The Element ID and Element Data Size are both encoded as a Variable Size Integer
 
 ## VINT_WIDTH
 
-Each Variable Size Integer begins with a VINT\_WIDTH which consists of zero or many zero-value bits. The count of consecutive zero-values of the VINT\_WIDTH plus one equals the length in octets of the Variable Size Integer. For example, a Variable Size Integer that starts with a VINT\_WIDTH which contains zero consecutive zero-value bits is one octet in length and a Variable Size Integer that starts with one consecutive zero-value bit is two octets in length. The VINT\_WIDTH MUST only contain zero-value bits or be empty.
+Each Variable Size Integer starts with a VINT\_WIDTH followed by a VINT\_MARKER. VINT\_WIDTH is a sequence of zero or more bits of value `0`, and is terminated by the VINT\_MARKER, which is a single bit of value `1`. The total number of bits (VINT\_WIDTH and VINT\_MARKER combined) is the number of octets in length of the Variable Size Integer.
```

Comment:
`the sum of VINT\_WIDTH and VINT\_MARKER` sounds better than `combined` which is vague mathematically. The total number in question is then more obvious.

Reply:
the sum of VINT_WIDTH and VINT_MARKER always equals `1` ;)

I changed to:
> The total length in bits of both VINT\_WIDTH and VINT\_MARKER is the total length in octets in of the Variable Size Integer.